src/pages/publication/PublicationFormValidationSchema.ts

```diff
@@ -1,11 +1,5 @@
 import * as Yup from 'yup';
```

Comment:
Lagde ny sak for å forenkle hele skjemaet her litt, og gjenbruke litt mer: https://unit.atlassian.net/browse/NP-1533?atlOrigin=eyJpIjoiYjIyZDIyYWY0OTMyNDZmNWIyM2Y4YWIwYzEyN2I0ZWQiLCJwIjoiaiJ9